frontend/src/lib/k8s/apiProxy.ts

```diff
@@ -91,7 +93,10 @@ async function refreshToken(token: string | null) {
     return;
   }
 
-  console.debug('Refreshing token');
+  if (isDebugVerbose('k8s/apiProxy@refreshToken')) {
+    console.debug('k8s/apiProxy@refreshToken', 'Refreshing token');
+  }
```

Comment:
Seems like all uses of `isDebugVerbose` are also printing a message with the same prefix as the param as the former function, so maybe this should be turned into a `printVerboseDebug('k8s/apiProxy@refreshToken', 'Refreshing token')`, so it internally does the check already?

Reply:
This trade off was done to give better log messages. If they are wrapped in a helper function then the log message says it is from the file where the helper is. This way the log message includes the file and line number where the debug log is, and stepping into the debugger is easier too.